Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
@NeerajW Yes, Having only OnGazeEnter/Leave is limiting. There is another class ScaleObjectMessageReceiver that illustrates how simple message handlers can be added to different objects to enable interesting behaviors in a modular way. For example, to adjust object size, I now simply need to drop FocusedObjectKeywordManager into the scene and add ScaleObjectMessageReceiver onto my objects. I could see a growing library of these simple behaviors added to HoloTookit in the future.